app/common/services/maps.js

```diff
@@ -62,7 +62,23 @@ function (
         layer.options = layer.layerOptions;
     });
 
+    // Icon configuration
+    function pointIcon(feature, size, className) {
+        return L.divIcon({
+            className: 'custom-map-marker ' + className,
+            html: '<svg class="iconic" style="fill:#' + feature.properties.color + ';"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="../../img/iconic-sprite.svg#map-marker"></use></svg><span class="iconic-bg" style="background-color:#' + feature.properties.color + ';""></span>',
```

Comment:
Could we sanitize the color value before injecting it into HTML? (just a regex for alpha numberic chars should do)